src/biome/text/configuration.py

```diff
@@ -160,7 +160,6 @@ class TrainerConfiguration:
     optimizer
     validation_metric
     patience
-    shuffle
```

Comment:
this param has been removed from Trainer configuration for 1.0. Same thing as with batch_sampler DataLoaders https://github.com/allenai/allennlp/issues/3966